js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -641,13 +645,6 @@ export const AuctionCard = ({
                     return;
                   }
 
-                  try {
-                    await update();
-                  } catch (e) {
-                    console.error('update (post-sendRedeemBid)', e);
-                    return;
-                  }
```

Comment:
Removing since accounts are update through the patchState 

https://github.com/holaplex/metaplex/pull/73/files#diff-2a3375edf8ebb5f1d006c10473dd5905f49f1761c3055caf7d7997f013adabb7L594